### PR TITLE
feat(docs-site): mainnet docs

### PR DESCRIPTION
**Pull request by RogerLamTd**

### CODE REVIEW

File: packages/docs-site/src/content/docs/network-reference/mainnet-addresses.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+---
+title: Mainnet Addresses
+description: Network reference page describing various important addresses on Taiko.
+---
+
+## Ethereum contracts
+
+| Contract Name (Shared) | Address                                      |
```
Comment (dantaik) [inline packages/docs-site/src/content/docs/network-reference/mainnet-addresses.md:8]: should we add an ENS column? See:
https://github.com/taikoxyz/taiko-mono/blob/main/packages/protocol/deployments/mainnet-contract-logs-L1.md

**Comment by dantaik** [comment]
We need to add the following explorer for mainnet:

- `taikoscan.io` (Block explorer by Etherscan)
- `taikoscan.network` (Block explorer by Router scan)
- `taiko.socialscan.io`  (Block explorer by SocialScan)

A couple of days later:
- `taiko.socialscan.io` renamed to `taikoscan.net`
- `taiko.wtf` (BCR explorer from zkPool)

Screenshot 2024-05-26 at 12 50 25

File: packages/docs-site/src/content/docs/network-reference/mainnet-addresses.md (hunk 1 above)
Comment (RogerLamTd) [reply packages/docs-site/src/content/docs/network-reference/mainnet-addresses.md]: added

**Comment by RogerLamTd** [comment]
> We need to add the following explorer for mainnet:
> 
> * `taikoscan.io` (Block explorer by Etherscan)
> * `taikoscan.network` (Block explorer by Router scan)
> * `taiko.socialscan.io`  (Block explorer by SocialScan)
> 
> A couple of days later:
> 
> * `taiko.socialscan.io` renamed to `taikoscan.net`
> * `taikonow.xyz` (BCR explorer from zkPool)
> 
> Screenshot 2024-05-26 at 12 50 25

I added them to rpc-configurations, if we want to add it to the main website maybe @bennettyong can help?

**Review by dionysuzx — APPROVED** [review]
verified all addresses. LGTM.

**Review by dantaik — APPROVED** [review]

**Review by bennettyong — APPROVED** [review]

**Review by dionysuzx — APPROVED** [review]

**Pull request merged**